src/Microsoft.DotNet.Interactive.Http.Parsing/Parsing/HttpRequestParser.cs

```diff
@@ -453,7 +453,8 @@ private void ConsumeCurrentTokenInto(HttpSyntaxNode node)
         {
             HttpUrlNode? node = null;
 
-            while ((CurrentToken?.Kind is HttpTokenKind.Word or HttpTokenKind.Punctuation) || (IsAtStartOfEmbeddedExpression()))
+            var significantToken = GetNextSignificantToken()?.Text.ToLowerInvariant();
+            while (((CurrentToken?.Kind is HttpTokenKind.Word or HttpTokenKind.Punctuation) || (CurrentToken is { Kind: HttpTokenKind.Whitespace } && significantToken is not "http/" or "https/") || (IsAtStartOfEmbeddedExpression())))
```

Comment:
The `/` char should probably be omitted from the string since it will not be in the same token as the `http` or `https` word.
```suggestion
            while (((CurrentToken?.Kind is HttpTokenKind.Word or HttpTokenKind.Punctuation) || (CurrentToken is { Kind: HttpTokenKind.Whitespace } && significantToken is not "http" or "https") || (IsAtStartOfEmbeddedExpression())))
```